ansible-tower-cli/ansible-tower-cli-task.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: ansible-tower-cli
+spec:
+  inputs:
+    params:
+      - name: SSLVERIFY
+        description: Disable tower ssl verification
+        default: "false"
+        type: string
+      - name: ARGS
+        description: The tower-cli commands to tun
+        type: array
+        default:
+          - "--help"
+      - name: HOST
+        description: The Ansible Tower host
+        type: string
+        default: ""
+      - name: tower-secret
+        description: The Ansible Tower secret with credentials
+        type: string
+        default: "tower-creds"
+  steps:
+    - name: config
+      image: quay.io/rcmendes/ansible-tower-cli:latest
+      command: 
```

Comment:
Could be done in a follow-up (most likely should) : this could use the [`script`](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#step-script) feature :stuck_out_tongue:

Reply:
Will change this and test, and open another issue and submit a PR is that okay?